tests/visual/components/loader/loader.css

```diff
@@ -0,0 +1,358 @@
+* {
```

Comment:
Ooh, this is making me think Vellum should be supplied as its own stylesheet in the test file.